mvp_example/src/main/java/com/distillery/android/blueprints/mvp/adapter/ToDoListDiffCallback.kt

```diff
@@ -0,0 +1,14 @@
+package com.distillery.android.blueprints.mvp.adapter
+
+import androidx.recyclerview.widget.DiffUtil
+import com.distillery.android.domain.models.ToDoModel
+
+class ToDoListDiffCallback : DiffUtil.ItemCallback<ToDoModel>() {
+    override fun areItemsTheSame(p0: ToDoModel, p1: ToDoModel): Boolean {
```

Comment:
It's always better to give variables full names
```suggestion
    override fun areItemsTheSame(model: ToDoModel, otherModel: ToDoModel): Boolean {
```